understanding/22/target-size-minimum.html

```diff
@@ -92,7 +100,7 @@ <h2>Intent of Target Size (Minimum)</h2>
     </section>
     <section id="benefits">
       <h2>Benefits of Target Size (Minimum)</h2>
-      <p>Having targets with sufficient target spacing can help all users who may have difficulty in confidently targeting or operating small controls. Users who benefit include, but are not limited to:</p>
+      <p>Having targets with sufficient size, or failing that, sufficient target spacing can help all users who may have difficulty in confidently targeting or operating small controls. Users who benefit include, but are not limited to:</p>
```

Comment:
"size, or failing that, sufficient target spacing can" might benefit from an extra comma or perhaps some hyphens "size – or failing that, sufficient target spacing – can"